csharp/ql/lib/semmle/code/csharp/Generics.qll

```diff
@@ -110,9 +110,10 @@ private string getTypeArgumentsToString(ConstructedGeneric cg) {
     strictconcat(Type t, int i | t = cg.getTypeArgument(i) | t.toStringWithTypes(), ", " order by i)
 }
 
+language[monotonicAggregates]
```

Comment:
Could we add comments stating that this is merely a precaution and that we expect he concat to only concat a single name.